### PR TITLE
[B360M-6] Refactoring documentation to point to the new Genesys docum…

### PR DESCRIPTION
**Pull request by OmerRahmanyGenesys**
…entation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+.DS_Store
```
Comment (ElizaSapir) [inline .gitignore:6]: @OmerRahmanyGenesys  please remove .DS_Store

**Review by ElizaSapir — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (OmerRahmanyGenesys) [reply .gitignore:6]: @ElizaSapir  
Don't we need to add it to the gitignore?
https://pineco.de/snippets/globally-gitignore-the-ds_store-file/

Comment (ElizaSapir) [reply .gitignore:6]: yes, we supposed to have it under `gitignore` like in all other repos we own.

Comment (OmerRahmanyGenesys) [reply .gitignore:6]: But this change is adding it to the gitignore so i don't understand the comment?
What should be done?

**Pull request merged**